CancelBooking UseCase/CancelBookinng.java

```diff
@@ -86,6 +90,8 @@ public boolean exists(String roomNumber, String cpr) {
 }
 // Business Layer
 // Provides methods for reservation management, including adding, canceling, and displaying reservations.
+
+
```

Comment:
```suggestion
you can improve the "processRefund" method to handle cases like no-show or cancellations after check-in.
```
```suggestion
instead of having a method handles both logic and user input/output (like cancelReservation), separate out logic from user-facing code
```